ExternalProjectDependency.cmake

```diff
@@ -447,16 +447,37 @@ function(_sb_get_external_project_arguments proj varname)
 
   set(_ep_arguments "")
 
-  # Automatically propagate CMake options
-  foreach(_cmake_option IN ITEMS
-    CMAKE_EXPORT_COMPILE_COMMANDS
-    CMAKE_JOB_POOL_COMPILE
-    CMAKE_JOB_POOL_LINK
-    CMAKE_JOB_POOLS
+  # Option CMAKE_FIND_USE_PACKAGE_REGISTRY was introduced in CMake 3.16
+  if(CMAKE_VERSION VERSION_GREATER "3.15")
```

Comment:
Version `3.15.1` will compare as greater than `3.15`, but does not have the new variable.  You could use `NOT VERSION_LESS 3.16`.  Or, if the minimum required version of CMake is at least 3.7, then `VERSION_GREATER_EQUAL 3.16`.

Reply:
Good catch. Thanks for the review :pray: